### ISSUE TITLE
Make @since a warning and try and omit it from the themes/ directory, or even better, specifically wd_s themes

### ISSUE THREAD

**Issue by aubreypwd** [WebDevStudios/WDS-Coding-Standards#39]
Per an engineering call on 3/7/18, @gregrickaby decreed :P that we remove `@since` requirements in wd_s/themes. This will omit wd_s/themes from being required (or even warned) about `@since`, and only make `@since` a warning outside of wds_s/themes.

# Why a Warning (outside of themes)

In being a warning, it suggests that it is not ALWAYS required, this will allow projects (or even plugins) that aren't doing any versioning to allow WDSCS to pass if we ever do any CS integration in deployments going forward.

Basically, `@since` is a very grey-area topic from project to project.
